index.js

```diff
@@ -148,3 +148,31 @@ for (let i = 0; i < menuLis.length; i += 1) {
     menu.classList.replace('menu', 'invisible');
   });
 }
+
+function isLower(str) {
+  return !(/[a-z]/.test(str) && !/[A-Z]/.test(str));
+}
+
+const form = document.querySelector('.form');
```

Comment:
- [ ] Great job so far :+1:  however, it would be better to move the `class="form" ` to the `form ` tag instead of the `section` tag, the submit button is always related to the form and not the section